packages/server-wallet/src/__test__/create-channel/direct-funding.test.ts

```diff
@@ -38,8 +38,14 @@ it('Create a fake-funded channel between two wallets ', async () => {
 
   const allocation: Allocation = {
     allocationItems: [
-      {destination: participantA.destination, amount: BigNumber.from(1).toHexString()},
```

Comment:
There seem to be prettier configuration differences between my local setup and @snario's. And both pass CI?

Reply:
Either before or after is valid when opening this file from VSCode on my machine, both when opening from the `monorepo` root as well as from the `packages/server-wallet` package.

This is also valid:
```
      {destination: participantA.destination, amount: BigNumber.from(1).toHexString()},
      {
        destination: participantB.destination,
        amount: BigNumber.from(1).toHexString(),
      },
```

This is one of the only times where prettier doesn't choose a unique representation of a bit of code. As a sanity check, you didn't add any newline characters and then formatted, did you?

Reply:
That is good to know. There is a chance I:
1. edited the value of `allocation` and saved.
1. edited the value of `allocation to match original and saved.